### DISCUSSION TITLE
解析出来的参考文献是不是可以支持批量复制以及导出bib文件?

### DISCUSSION THREAD

**Discussion by henusys** [MuiseDestiny/zotero-reference#124] [Ideas]
这个参考文献解析的功能真的非常厉害, 也非常的好用, 那么是不是可以直接把这些参考文献批量复制并且可以直接导出bib文件,这样非常方便处理. 

另外还有两个小小的建议. 
1, 当前点击参考文献的加号按钮的时候,参考文献会被添加到所有与当前文献有关的文件夹内. 比如说当前的pdf同时存在文献库的三个分类 文件夹中, 那么 当我选择添加文献时,新的文献会被同时保存到与之有关的三个分类文件夹中. 其实我是希望可以添加到某个分类问价中就可以了. 
2. 在打开的PDF中, 已经解析过参考文献的会被保存在某个json文件中. 那么当这个PDF关闭以后再次打开(刷新参考文献), 按道理说直接读取本地参考文献就可以了,这个速度应该是非常快的, 但是不知道问什么感觉参考文献已经读取完毕了,软件仍然是会卡顿一段时间. 这个显现对于阅读综述类文章尤其明显. 综述文章的文献数量有可能在500个左右, 此时软件就会出现大约10s的卡顿. 不知道在这方面您是不是有什么可以改进的地方. 

zotero-reference是一个非常有意义的软件, 非常感谢您的无私贡献.

**Comment by MuiseDestiny** [comment]
谢谢

>  其实我是希望可以添加到某个分类问价中就可以了.

试试ctrl+点击加号

> 软件仍然是会卡顿一段时间

搜索本地匹配，因为本地文献可能会随时变化，所以每次会有一个匹配时间，首选项最后一个框设置为1，可以关闭这个功能

**Comment by henusys** [comment]
经过今天的测试, 发现Zoter Reference 插件的性能还不够稳定.  开启这个插件后,经常会导致阅读PDF时卡死的现象. 即便参考文献已经更新完毕, 但是当我阅读PDF,做一些笔记的时候就会导致Zotero无响应. 当关闭这个插件之后,问题就会消失. 是不是有可能在阅读PDF且当鼠标点击PDF页面时, 触发了这个插件更新某些数据更新的操作, 从而导致了异常的CPU资源占用?  



***@***.***
 
发件人： Polygon
发送时间： 2023-04-10 15:05
收件人： MuiseDestiny/zotero-reference
抄送： henusys; State change
主题： Re: [MuiseDestiny/zotero-reference] 解析出来的参考文献是不是可以支持批量复制以及导出bib文件? (Discussion #124)
谢谢
其实我是希望可以添加到某个分类问价中就可以了.
试试ctrl+点击加号
软件仍然是会卡顿一段时间
搜索本地匹配，因为本地文献可能会随时变化，所以每次会有一个匹配时间，首选项最后一个框设置为1，可以关闭这个功能

**Reply by MuiseDestiny** [reply]
> 经过今天的测试, 发现Zoter Reference 插件的性能还不够稳定. 开启这个插件后,经常会导致阅读PDF时卡死的现象. 即便参考文献已经更新完毕, 但是当我阅读PDF,做一些笔记的时候就会导致Zotero无响应. 当关闭这个插件之后,问题就会消失. 是不是有可能在阅读PDF且当鼠标点击PDF页面时, 触发了这个插件更新某些数据更新的操作, 从而导致了异常的CPU资源占用? ***@***.*** 发件人： Polygon 发送时间： 2023-04-10 15:05 收件人： MuiseDestiny/zotero-reference 抄送： henusys; State change 主题： Re: [MuiseDestiny/zotero-reference] 解析出来的参考文献是不是可以支持批量复制以及导出bib文件? (Discussion #124) 谢谢 其实我是希望可以添加到某个分类问价中就可以了. 试试ctrl+点击加号 软件仍然是会卡顿一段时间 搜索本地匹配，因为本地文献可能会随时变化，所以每次会有一个匹配时间，首选项最后一个框设置为1，可以关闭这个功能 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you modified the open/close state.Message ID: ***@***.***>

首选项把link下面的功能关了

**Comment by henusys** [comment]
哈哈, 还真是这个问题! 谢谢~



***@***.***
 
发件人： Polygon
发送时间： 2023-04-10 22:54
收件人： MuiseDestiny/zotero-reference
抄送： henusys; State change
主题： Re: [MuiseDestiny/zotero-reference] 解析出来的参考文献是不是可以支持批量复制以及导出bib文件? (Discussion #124)
经过今天的测试, 发现Zoter Reference 插件的性能还不够稳定. 开启这个插件后,经常会导致阅读PDF时卡死的现象. 即便参考文献已经更新完毕, 但是当我阅读PDF,做一些笔记的时候就会导致Zotero无响应. 当关闭这个插件之后,问题就会消失. 是不是有可能在阅读PDF且当鼠标点击PDF页面时, 触发了这个插件更新某些数据更新的操作, 从而导致了异常的CPU资源占用? @.*** 发件人： Polygon 发送时间： 2023-04-10 15:05 收件人： MuiseDestiny/zotero-reference 抄送： henusys; State change 主题： Re: [MuiseDestiny/zotero-reference] 解析出来的参考文献是不是可以支持批量复制以及导出bib文件? (Discussion #124) 谢谢 其实我是希望可以添加到某个分类问价中就可以了. 试试ctrl+点击加号 软件仍然是会卡顿一段时间 搜索本地匹配，因为本地文献可能会随时变化，所以每次会有一个匹配时间，首选项最后一个框设置为1，可以关闭这个功能 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you modified the open/close state.Message ID: @.***>
首选项把link下面的功能关了
